scripts/test-ci.sh

```diff
@@ -6,8 +6,6 @@ set -euo pipefail
 # this is required for sonarcloud analysis
 if [ "${TRAVIS_NODE_VERSION}" = "10" ]; then
   echo 'Running tests with coverage and reporter'
-  # install `jest-sonar-reporter` here, because otherwise `yarn install` fails on nodejs
```

Comment:
Used to be true for Node.js 6, but was unnecessarily kept when the later was dropped in favour of Node.js 8.